airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py

```diff
@@ -212,21 +212,27 @@ def _property_can_store_secret(prop: dict) -> bool:
         A string, a number or an integer type can always store secrets.
         Objects and arrays can hold a secret in case they are generic,
         meaning their inner structure is not described in details with properties/items.
+        A field with a constant value can not hold a secret as well.
         """
         unsecure_types = {"string", "integer", "number"}
         type_ = prop["type"]
         is_property_generic_object = type_ == "object" and not any(
             [prop.get("properties", {}), prop.get("anyOf", []), prop.get("oneOf", []), prop.get("allOf", [])]
         )
         is_property_generic_array = type_ == "array" and not any([prop.get("items", []), prop.get("prefixItems", [])])
-        return any(
+        is_property_constant_value = bool(prop.get("const"))
```

Comment:
Wdyt about creating a `cannot_store_secret` list with this `is_property_constant_value`?

Reply:
Not sure I got your idea. Do you want to make a one item list? What should we do with it next?

Reply:
Yes, a one item list at the moment but that could grow if you detect other similar adjustments to make. Just for readability. Very minor suggestion, feel free to ignore if you don't think it's legit.

Reply:
I'm no sure those future items would be treated the same as this one. So I'd prefer to leave it as is for now